package bftbench;




message Response
{
	required bool BoolResponse = 1;
	optional string StringResponse = 2;
	repeated string ListResponse = 3;
}